init.sql

```diff
@@ -25,6 +25,38 @@ CREATE INDEX cl_items_node_idx on cl_items (node_idx);
 CREATE INDEX cl_items_leaf_idx on cl_items (leaf_idx);
 CREATE UNIQUE INDEX cl_items__tree_node on cl_items (tree, node_idx);
 
+CREATE TABLE backfill_items
+(
+    id          bigserial PRIMARY KEY,
+    tree        BYTEA  NOT NULL,
+    seq         BIGINT NOT NULL,
+    slot        BIGINT NOT NULL,
+    force_chk   bool
```

Comment:
Why store `id` in this table? Seems like this is just used to track gaps in the `tree` sequence numbers?

Reply:
I agree I do not think ID is needed for anything, I was just following the fact that all of the other tables had an ID as a primary key.